homeassistant/components/binary_sensor/hive.py

```diff
@@ -35,9 +35,24 @@ def __init__(self, hivesession, hivedevice):
         self.attributes = {}
         self.data_updatesource = '{}.{}'.format(self.device_type,
                                                 self.node_id)
-
+        self.uniqueid = '{}-{}'.format(self.node_id, self.device_type)
```

Comment:
Use `self._unique_id`.

Reply:
This has been updated and moved to a new PR